templates/core/index.html

```diff
@@ -6,127 +6,334 @@
 
 {% block content %}
 
-
-<section class="p-strip--suru is-dark">
+<section class="p-strip-suru-half-top">
   <div class="row u-equal-height u-vertically-center">
-    <div class="col-6">
-      <h1>Build secure IoT devices  <br> with Ubuntu Core</h1>
-      <p class="p-heading--four u-sv3">Everything you love about Ubuntu, locked down for security. Helping you make safer things &ndash; because we're all connected.</p>
+    <div class="col-7">
+      <h1>Secure embedded devices</h1>
+      <p class="p-heading--four u-sv3">Give your device its own app store</p>
+      <p>
+        <a class="p-button--positive" href="/core/smartstart">Get to market faster</a>
+      </p>
+      <p>
```

Comment:
```suggestion
      <p class="u-sv3">
```

I think there needs to be a bit more space,